publish-pypi.yml

```diff
@@ -0,0 +1,123 @@
+jobs:
+- job: linux_wheels
+  dependsOn: ${{ parameters.dependsOn }}
+  condition: succeeded()
+  pool: {vmImage: 'Ubuntu-16.04'}
+  steps:
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        pip install cibuildwheel==0.11.1
+        cibuildwheel --output-dir wheelhouse .
+    - task: PublishBuildArtifacts@1
+      inputs: {pathtoPublish: 'wheelhouse', artifactName: 'linux_wheels' }
+
+- job: macos_wheels
+  dependsOn: ${{ parameters.dependsOn }}
+  condition: succeeded()
+  pool: {vmImage: 'macOS-10.13'}
+  steps:
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        pip install cibuildwheel==0.11.1
+        cibuildwheel --output-dir wheelhouse .
+    - task: PublishPipelineArtifact@0
+      inputs: {targetPath: 'wheelhouse', artifactName: 'macos_wheels' }
+
+- job: windows_wheels
+  dependsOn: ${{ parameters.dependsOn }}
+  condition: succeeded()
+  pool: {vmImage: 'vs2017-win2016'}
+  steps:
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.6', architecture: x64}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.7', architecture: x64}}
+    - bash: |
+        python -m pip install --upgrade pip
+        pip install cibuildwheel==0.11.1
+        cibuildwheel --output-dir wheelhouse .
+    - task: PublishPipelineArtifact@0
+      inputs: {targetPath: 'wheelhouse', artifactName: 'windows_wheels' }
+
+- job: sdist
+  dependsOn: ${{ parameters.dependsOn }}
+  condition: succeeded()
+  pool: {vmImage: 'Ubuntu-16.04'}
+  steps:
+  - script: |
```

Comment:
arrgh haven't they fixed this yet :roll_eyes:

Reply:
Nope.